### ISSUE TITLE
Deploy fails with: error pulling image configuration, TLS handshake timeout

### ISSUE THREAD

**Issue by flinz** [SwissDataScienceCenter/renku#408]
After trying to deploy Renku, I got 
```bash
> make minikube-deploy                                                                                                                                                                          
pipenv run python scripts/minikube_deploy.py
Sending build context to Docker daemon  2.968MB
Step 1/13 : FROM node:8.11.1-alpine
8.11.1-alpine: Pulling from library/node
605ce1bd3f31: Pulling fs layer
0511902e1bcd: Pulling fs layer
343e34c41f87: Pulling fs layer
error pulling image configuration: Get https://production.cloudflare.docker.com/registry-v2/docker/registry/v2/blobs/sha256/e7/[ .....] : net/http: TLS handshake timeout
```

This can be solved by doing
```bash
> docker logout
> docker login
```

_Edit: The error returned intermittently._
I am on OSX with VirtualBox. Some additional google yielded this: https://forums.docker.com/t/docker-pull-intermittent-tls-handshake-timeout/35008
**TL;DR: disable automatic proxy in VirtualBox preferences (set to direct internet).**
This now seems to resolve the issue.

**Comment by leafty** [comment]
Hi, thanks for reporting this. I do not know if this is something that minikube users are likely to encounter, so it might take some time until we have it added to a troubleshooting doc.

I am closing the issue as it is related to virtualbox and not Renku.
